core/order_test.go

```diff
@@ -356,3 +358,27 @@ func TestOpenBazaarNode_CalculateOrderTotal(t *testing.T) {
 		t.Error("Calculated wrong order total")
 	}
 }
+
+func TestOpenBazaarNode_GetOrder(t *testing.T) {
+	node, err := test.NewNode()
+	if err != nil {
+		t.Error(err)
+	}
+
+	contract := factory.NewContract()
+	orderID, err := node.CalcOrderID(contract.BuyerOrder)
+	err = node.Datastore.Purchases().Put(orderID, *contract, pb.OrderState_AWAITING_PAYMENT, false)
+
+	orderResponse, err := node.GetOrder(orderID)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if orderResponse.Funded {
+		t.Error(errors.New("Should not be funded"))
+	}
+
+	if orderResponse.State != pb.OrderState_AWAITING_PAYMENT {
+		t.Error(errors.New("Expected state OrderState_AWAITING_PAYMENT"))
```

Comment:
It might be helpful to indicate what `orderResponse.State` actually was. Something like `expected attribute description to be (X), but was (Y)` would be ideal.

Reply:
done